tests/plugins/test_orchestrate_build.py

```diff
@@ -351,6 +369,8 @@ def test_orchestrate_build(tmpdir, caplog, config_kwargs,
                     namespace = {name}_namespace
                     """.format(name=cluster['name'])))
 
+    goarch = {'x86_64': 'amd64', 'ppc64le': 'ppc64'}
```

Comment:
I think it doesn't matter because this is just test data, but the GOARCH for 'ppc64le' is 'ppc64le' so we wouldn't bother including that key. A likely-looking dict would be `{'x86_64': 'amd64', 'aarch64': 'arm64'}`.

Reply:
well even if it is same it has to be in goarch, or we will need to change logic how we use it, that is used in group_manifest plugin as well

Reply:
No, definitely not:
https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/post_group_manifests.py#L236

This is particularly important, because we do not have a platform descriptor for ppc64le because it doesn't need one.

Reply:
ok will change how to use platform_descriptors and default to platform if not there

Reply:
Can we also remove `ppc64le` item to ensure the code is not requiring it?

Reply:
ok